src/sentry/backup/imports.py

```diff
@@ -441,8 +441,33 @@ def import_in_global_scope(
     Sentry instance, some behaviors in this scope are different from the others. In particular,
     superuser privileges are not sanitized. This method can be thought of as a "pure"
     backup/restore, simply serializing and deserializing a (partial) snapshot of the database state.
+    During the setup of a fresh Sentry instance, there are a couple of models that are automatically created.
+    The Sentry org, a Sentry team, and an internal project. During a global import, we want to avoid persisting
+    these default models and start from scratch.
     """
 
+    from django.core.management.color import no_style
+    from django.db import connection
+
+    from sentry.models.organization import Organization
+    from sentry.models.organizationmember import OrganizationMember
+    from sentry.models.project import Project
+    from sentry.models.projectkey import ProjectKey
+    from sentry.models.team import Team
+
+    models_to_delete = [Project, ProjectKey, Organization, OrganizationMember, Team]
```

Comment:
Imo, we should do this more completely, rather than guessing at models (ex: it is possible that the "default" models on a new install change in the future). Luckily, we already have a test function to do exactly this at `src/sentry/testutils/helpers.py::clear_database`: it creates "TRUNCATE ..." DB calls, fires them off in the correct order (so that dependencies are always deleted before the things that depend on them), and then resets the sequence counter.

We'll probably need copy this function and change its structure a bit, since it has a lot of `assume_test_silo` calls that are specifically intended to help with testing only. Luckily, we can ignore all of the RPC call-based synchronization between control and region models, since `Global` imports are only done on self-hosted instances (see comment below).

Reply:
done